lib/pos_blockchain.go

```diff
@@ -348,8 +355,17 @@ func (bc *Blockchain) validateBlockLeader(desoBlock *MsgDeSoBlock) error {
 // validateQC validates that the QC of this block is valid, meaning a super majority
 // of the validator set has voted (or timed out). Assumes ValidatorEntry list is sorted.
 func (bc *Blockchain) validateQC(desoBlock *MsgDeSoBlock, validatorSet []*ValidatorEntry) error {
```

Comment:
nit: rename boolean functions to use an `isXYZ` or `hasXYZ` naming pattern. It's also good to pass in the bare minimum data needed to perform the validation. In this case, we only need to pass in the QC and validator set. All good little things that make the function more self-documenting based on the signature alone.
```suggestion
func (bc *Blockchain) isValidQC(qc *QuorumCertificate, validatorSet []*ValidatorEntry) error {
```